frontend/src/components/ViewWrappers/ViewWrappers.js

```diff
@@ -33,11 +33,11 @@ const View = ({ children, withPadding }) => (
   </div>
 );
 
-const ViewSplitter = ({ children, omitOnMobile, withPadding }) => {
+const ViewSplitter = ({ children, omitOnMobile, withPadding, small }) => {
   const isBigScreen = useMediaQuery({ minWidth: 960 });
 
   return omitOnMobile && !isBigScreen ? false : (
-    <div className={`ViewSplitter ${withPadding && 'ViewSplitter--padded'}`}>
+    <div className={`ViewSplitter ${withPadding && 'ViewSplitter--padded'} ${small && 'ViewSplitter--small'}`}>
```

Comment:
Conditions like this `withPadding && 'ViewSplitter--padded'` always result to `className="undefined undefined"`.
i think you cound create a simple function as the class name resolver
`const funcName  = (condition, className) => condition ? className : ''`